Emby.Naming/Common/NamingOptions.cs

```diff
@@ -250,6 +250,8 @@ public NamingOptions()
                 },
                 // <!-- foo.ep01, foo.EP_01 -->
                 new EpisodeExpression(@"[\._ -]()[Ee][Pp]_?([0-9]+)([^\\/]*)$"),
+                // <!-- foo.E01., foo.e01. -->
+                new EpisodeExpression(@"[^\\/]*?()\.?[Ee]([0-9]+)\.([^\\/]*)$"),
```

Comment:
Why does this expression differ from the one above?

Reply:
The new one solves the case like
```
The.Legend.of.Condor.Heroes.2017.E07.V2.web-dl.1080p.h264.aac-hdctv.mkv
```
It's a very common name format in PT，this format offen like: xx.xxx.E01.xxxx.xxx
but the old can't support this format

Reply:
I'm asking why your new expression starts with `[^\\/]*?()\.?` when the other one starts with `[\._ -]()`.

Wouldn't changing the expression above to `[\._ -]()[Ee][Pp]?_?([0-9]+)([^\\/]*)$` work just as well (I added `?` after `[Pp]`)?

Reply:
If you use your way, this test case will run failed
https://github.com/jellyfin/jellyfin/blob/0c9b64de4be633f2a4f47244996c8a25bbb0db45/tests/Jellyfin.Naming.Tests/TV/EpisodeNumberTests.cs#L49
this case will be recognized as episode 15

Reply:
I'm not sure what `2009x03-E15` is supposed to mean. Season 2009 Episode 3, but what is E15?

Reply:
I do not know either what is E15, so I add new one. Do you have a better suggestion?

Reply:
I would read it as year 2009, season 3, episode 15 or something to that affect